deploy/lvm-operator.yaml

```diff
@@ -246,6 +246,118 @@ status:
   conditions: []
   storedVersions: []
 
+
+##############################################
+###########                       ############
+###########     LVMNode CRD       ############
+###########                       ############
+##############################################
+
+# LVMNode CRD is autogenerated via `make manifests` command.
+# Do the modification in the code and run the `make manifests` command
+# to generate the CRD definition
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
we can start using `v1` CRDs version ..`v1beta` going to be deprecated in k8s 1.22

Reply:
Yeah, I had initially configured the same, then saw that other crds (LVMSnapshot, LVMVolume etc) are using v1beta1. I think, we can take this up as part of separate pull request as it'll required changes to other crds as well.

Reply:
Yeah, I had initially configured the same, then saw that other crds (LVMSnapshot, LVMVolume etc) are using v1beta1. I think, we can take this up as part of separate pull request as it'll required changes to other crds as well.